ci-operator/step-registry/dpf/hypervisor/network-tests/dpf-hypervisor-network-tests-commands.sh

```
@@ -1,159 +1,14 @@
-
-fi
-
-# Parse the log files, may need to scp to container running ssh cmds and process the
-# output file and exit accordingly
-
-if [ "${NETWORK_TESTS_RESULT}" == "PASS" ]; then
-  exit 0
-fi
-
-exit 1
+export TFT_SERVER_NODE=$(oc get nodes --no-headers | grep worker-dpu | awk 'NR==1 {print $1}')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)